src/TerminalFighter/Title/Title.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Title.h"
```

Comment:
It's unlikely we'll keep this class in the future, not generic enough, let's rename it to text box.